src/legacy/core_plugins/kibana/public/dashboard/listing/dashboard_listing.js

```diff
@@ -26,6 +26,11 @@ import { EuiLink, EuiButton, EuiEmptyPrompt } from '@elastic/eui';
 
 import { TableListView } from './../../table_list_view';
 
+import chrome from 'ui/chrome';
+import { UserAPIClient } from '../../../../../../../x-pack/legacy/plugins/security/public/lib/api';
```

Comment:
We can't rely on X-Pack (Elastic License) code from within an OSS (Apache 2.0) file. This file needs to be able to execute without having **any** knowledge of our security or spaces plugins, and these changes would effectively couple the OSS dashboard listing page to Elastic licensed features.